tests/light/src/axosyslog_light/fixtures.py

```diff
@@ -197,7 +197,7 @@ def log_message():
 
 @pytest.fixture
 def version(syslog_ng):
-    return syslog_ng.get_version()
+    return syslog_ng.get_config_version()
```

Comment:
AxoSyslog's version and config version is not the same, why do we use the config version getter in the version fixture?

Reply:
Yes, that is correct. Thanks for this. I will fix this part.

Reply:
fixed